Demo/Demo/SwiftUIComponents/PayPalWebPayments/PayPalWebButtonsView.swift

```diff
@@ -10,43 +10,52 @@ struct PayPalWebButtonsView: View {
 
     var body: some View {
         VStack {
-            VStack(alignment: .center, spacing: 16) {
-                HStack {
-                    Text("Checkout with PayPal")
-                        .font(.system(size: 20))
-                    Spacer()
+            VStack(alignment: .center, spacing: 40) {
+                PayPalButton.Representable(color: .gold, size: .mini) {
+                    payPalWebViewModel.paymentButtonTapped(funding: .paypal)
```

Comment:
Looks like these lines were unintentionally added likely due to a merge conflict from #227 (unless this branch hasn't been updated with `main` since that PR was merged). We should update this PR as right now this view does not render correctly:
![Simulator Screenshot - iPhone 15 - 2024-01-08 at 09 00 03](https://github.com/paypal/paypal-ios/assets/20733831/663bab4d-422b-4108-aead-172dbf33d652)

Reply:
Yea I'm not sure what happened here. I haven't updated my branch with the latest from `main` so I'll do that and get the issue resolved